memdown.js

```diff
@@ -127,9 +132,15 @@ MemDOWN.prototype._open = function (options, callback) {
   })
 }
 
-MemDOWN.prototype._put = function (key, value, options, callback) {
-  if (typeof value === 'undefined' || value === null) value = ''
+MemDOWN.prototype._serializeKey = function (key) {
+  return key
+}
+
+MemDOWN.prototype._serializeValue = function (value) {
+  return value == null ? '' : value
```

Comment:
why would we not store `null`?

Reply:
This null behavior is the same as the default `_serializeValue` implementation of `abstract-leveldown`. I wanted to keep this as-is for now, and attack the null/undefined question later (across the board).

Main reason not to store null is because `null` signifies end in Node.js streams. More at https://github.com/Level/abstract-leveldown/issues/120.

Reply:
true, I remember now :)